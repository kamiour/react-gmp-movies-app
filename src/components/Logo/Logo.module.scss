@import 'src/scss/variables';

.logoLink {
  font-weight: 500;
  font-size: 20px;
  line-height: 24px;
  color: $color_wild_watermelon;
  text-decoration: none;

  .logoLinkBold {
    font-weight: 900;
  }
}
