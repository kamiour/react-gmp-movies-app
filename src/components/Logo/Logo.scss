@import 'src/scss/variables';

.logo-link {
  font-weight: 500;
  font-size: 20px;
  line-height: 24px;
  color: $color_wild_watermelon;
  text-decoration: none;

  .logo-link-bold {
    font-weight: 900;
  }
}
