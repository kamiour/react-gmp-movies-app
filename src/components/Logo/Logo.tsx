// import './Logo.scss';

const Logo = () => (
  <a className="logo-link" href="/">
    <span className="logo-link-bold">netflix</span>roulette
  </a>
);

export default Logo;
