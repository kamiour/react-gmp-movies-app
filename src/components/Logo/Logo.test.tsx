import { render } from '@testing-library/react';
import Logo from './Logo';

describe('Logo', () => {
  it('should match Logo snapshot', () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
