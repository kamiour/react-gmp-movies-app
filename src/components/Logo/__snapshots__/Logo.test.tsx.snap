// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match Logo snapshot 1`] = `
<DocumentFragment>
  <a
    class="logoLink"
    href="/"
  >
    <span
      class="logoLinkBold"
    >
      netflix
    </span>
    roulette
  </a>
</DocumentFragment>
`;
